card/lib/card/format/render.rb

```diff
@@ -73,11 +73,13 @@ def view_cache_setting view
       end
 
       def stub_render cached_content
-        expand_stubs cached_content do |stub_hash|
+        result = expand_stubs cached_content do |stub_hash|
           prepare_stub_nest(stub_hash) do |stub_card, mode, options, override|
             with_nest_mode(mode) { nest stub_card, options, override }
           end
         end
+        puts "STUB IN RENDERED VIEW: #{result}" if result =~ /stub/
```

Comment:
I see my code made it in :)

I want to make this a bit noisier and leave this around for a bit until we stop seeing stubs where we don't want them.